packages/video_player/video_player_web/lib/video_player_web.dart

```diff
@@ -131,7 +131,6 @@ class VideoPlayerPlugin extends VideoPlayerPlatform {
 
   @override
   Future<Duration> getPosition(int textureId) async {
-    _videoPlayers[textureId].sendBufferingUpdate();
```

Comment:
Nit: This change makes sense to me, but I'd add a note to the CHANGELOG about it. "The videoplayer won't fire a buffering update event immediately when getting the playhead position" or something like that. This might be breaking for some (even if the buffering is sent immediately afterwards, we may be changing the order in which events are sent to the app). This is probably not a breaking change that's going to affect many people, but in event-driven apps, it might confuse users.

Reply:
Some context on why I removed this; setting the player position would result in a buffering update event being sent immediately, this would happen before any buffering start event and didn't really make sense to be like that, especially since internally the event listeners already send buffering update events.

I could add this back in (same on Android) but the events would be `bufferingUpdate`, `bufferingStart`, `bufferingUpdate` (as many times as applicable), `bufferingEnd` - which doesn't seem right to me 🤔 

I'm on the fence on whether I should just revert this change and not worry about those events in the scope of this PR vs having a breaking change.

Thoughts, should I just revert or are we ok with having a breaking change here?

Reply:
Reverted this change for now, could address it in another PR if necessary.